### DISCUSSION TITLE
Visualization position relative to mouse cursor position

### DISCUSSION THREAD

**Discussion by rsyqvthv** [mulaRahul/keyviz#47] [General]
I usually make partical screen recording which following my mouse for  some howto vids. (like [this](https://www.bandicam.com/how-to/around-mouse/))

keyviz is beautiful and slim, 
but it only good for whole screen recording,
when I do the mouse following screen recording, 
the keyviz's keystroke is out of the recording area.

So I would like to have the keystrokes can be "dymatically alignment to mouse" option, 
which basically the keystrokes follows the mouse and can be adjust the distance to mouse (to right, or to bottom).

or, bit imprefect but acceptable, the keystrokes window can be drag to reloate.

I've looking for this for long time,
but none of virsual keystrokes software have this function.

would you like to add this to your gorgeous software?

**Comment by mulaRahul** [comment]
Well, it's a really niche-specific feature. Most of the presentations/recordings are done in full-screen. But it sure will be helpful in situations like yours. Though, the same can only be possible after mouse support is added to the project.

**Reply by rsyqvthv** [reply]
Thanks for replying, :)

yes, most of the presentations are done in full-screen,
I agree indead, that's why I can't check most of them on the mobile,
as they are too small, I can't even follow their mouse movement.

some of the presentations do use zoom in to show detials, 
but it do needs post-editing, 
to zoom in and add keystroke one by one manually on each scene,

that workflow is quite time consuming,
I think that's why most of the full screen recording vids take the easiest way:
just speak out the key they input,
but doing so loses the intuitiveness of the video itself.

that's why I think, 
a little mouse following virtual keystroke software will be the game changer.
with it, 
I can always recording detailed window straight away, 
has bigger mouse to easily catch up, 
and every combine key I input to easily tell, 
even more:
has more chance that I don't need any post-editing any more!

this function will make keyviz unique,
as I researched before,
there isn't any similar tools manipulate this yet.

hopefully, youtube will have more mobile friendly presentations soon, 
which all be with **the keyviz** key strokes.

;)

**Reply by mulaRahul** [reply]
Sounds good! I will start by adding mouse support (mouse-position highlight and click events) and then implement the relative positioning feature. 

_Maybe_ a hotkey to switch between relative (_to mouse cursor_) and absolute (_to screen_) positioning as well? You're welcome to add more suggestions on the same.
